.github/workflows/automated_release.yml

```diff
@@ -61,15 +61,18 @@ jobs:
           docker.pkg.github.com/${IMAGE_NAME}:${BRANCH_NAME}
           python -m pytest --no-cov -vx
 
-      - name: Set release tag
+      - name: Set release name
         # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
-        run: echo "RELEASE_VERSION=$(printf -- '%s%s\n' $(cat .release-version) $([ ${BRANCH_NAME} = "develop" ] && printf -- '-%s-develop' ${GITHUB_RUN_ID} || echo ""))" >> $GITHUB_ENV
+        run: echo "RELEASE_NAME=$(printf -- '%s%s\n' $(cat .release-version) $([ ${BRANCH_NAME} = "develop" ] && printf -- '-%s-develop' ${GITHUB_RUN_ID} || echo ""))" >> $GITHUB_ENV
+
+      - name: Set release tag
+        run: echo "RELEASE_TAG=v$RELEASE_NAME" >> $GITHUB_ENV
 
       - name: Create release
         uses: ncipollo/release-action@v1.8.8
         with:
-          name: ${{ env.RELEASE_VERSION }}
-          tag: v${{ env.RELEASE_VERSION }}
+          name: ${{ env.RELEASE_NAME }}
```

Comment:
So I would give the release name a `v` as well as the tag name, just so there are `v`s everywhere and it's unambiguous if people don't know exactly which string they're meant to use in the Ansible command.

Reply:
I think I'd rather keep consistency with non-Docker repos for now.  Thanks for the suggestion though.  If it still presents a problem we can change it.